docs/pages/billing/faq.mdx

```diff
@@ -0,0 +1,119 @@
+---
+title: Frequently Asked Questions (FAQs) about payments, plans, and billing
+sidebar_title: FAQs
+description: A reference of commonly asked questions on Expo Application Services (EAS) plans, billing, and payment.
+---
+
+This page covers frequently asked questions about plans, billing, and payment for Expo Application Services (EAS).
+
+## Plans
+
+### How can I update my plan?
+
+To update your Organization account's plan, make sure that you have either [**Owner** or **Admin** role privilege](/accounts/working-together/#manage-access). For a Personal account, you always have an **Owner** role.
+
+After verifying the role, see [Upgrade to a new plan](/billing/manage/#upgrade-to-a-new-plan) to upgrade or [Downgrade a plan](/billing/manage/#downgrade-a-plan) to downgrade the existing plan.
+
+### How can I cancel a plan?
+
+See [Cancel a plan](/billing/manage/#cancel-a-plan) for more information.
+
+### What if I subscribe from the wrong account?
+
+If you've subscribed to a plan from the wrong account:
+
+- From the Expo dashboard's navigation menu, under **Account**, switch to the account from which you intend to subscribe.
+- Go to [**Billing**](https://expo.dev/settings/billing) and under **Plan**, [follow the steps to subscribe to the right plan](/billing/manage/#upgrade-to-a-new-plan).
+- From the wrong account, go to **[Receipts](https://expo.dev/accounts/[account]/settings/receipts)** and initiate a [request for a refund](/billing/invoices-and-receipts/#request-a-refund).
+
+### I am on a Free plan and only need a few extra builds
+
+If you are on a Free plan and have completed your monthly quota of free builds, upgrade to the [On-demand plan](https://www.notion.so/Subscriptions-Plans-and-Add-ons-3f928a4b5a944f918cd7cc5a17c91005?pvs=21). It offers a pay-as-you-go option for EAS Build and the same limits as the Free plan for EAS Update. Once your requirements are fulfilled, you can always downgrade to the Free from the On-demand plan.
+
+To upgrade from the Free to the On-demand plan, see [Upgrade to a new plan](/billing/manage/#upgrade-to-a-new-plan).
+
+To downgrade from an On-demand to a Free plan, see [Cancel a plan](/billing/manage/#cancel-a-plan).
+
+## Billing
+
+### What does a billing cycle start for a plan?
+
+For the Free plan, the billing cycle starts on the first day of the calendar month.
+
+For all other paid plans, the billing cycle starts from the subscription date of that plan.
+
+### How do I update my billing information?
```

Comment:
Nit: we should add "tax ID" here too since it's a common question: 

`How do I update my billing information or add a tax ID?`